src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/package-info/test4/package-info.java

```diff
@@ -0,0 +1,5 @@
+//for test that there's warning when single line comment isn't separated from PACKAGE_DEF by line
+//Should cause violation after this comment
+package com.puppycrawl.tools.checkstyle.checks.whitespace.test4;
```

Comment:
com/puppycrawl/tools/checkstyle/checks/whitespace/package-info/test4/package-info.java

vs 

package com.puppycrawl.tools.checkstyle.checks.whitespace.test4;

Where is "package-info" subpackage ? please make it and make it as "packageinfo". No "-" in package names.